packages/core/nuxt-theme-module/theme/components/InstagramFeed.vue

```diff
@@ -1,20 +1,24 @@
 <template>
-  <SfSection title-heading="Share Your Look" subtitle-heading="#YOURLOOK" class="section">
+  <SfSection title-heading="Share Your Look" subtitle-heading="#YOURLOOK" class="title">
     <div class="grid grid-images">
       <div class="grid__row">
         <div class="grid__col">
-          <SfImage src="/homepage/imageA.webp" :width="486" :height="486">katherina_trn</SfImage>
+          <SfImage class="smartphone-only" src="/homepage/imageAm.webp" :width="160" :height="160">katherina_trn</SfImage>
```

Comment:
if you hide images only with CSS they will still load (so both versions will ALWAYS load) - we need `v-if` here